provision/acc_provision/flavors.yaml

```diff
@@ -103,6 +103,8 @@ flavors:
         kubectl: oc
         system_namespace: aci-containers-system
       aci_config:
+        port_range:
+          - [2379, 2379]
```

Comment:
Its great that you were incorporated this change in response to the review comment. But since we are doing this, we should also enhance this to be able to specify other attributes of the filter entry. Can we do something like this:
aci_config:
    ports:
        - name: openshift-svc-catalog
           range: [2379, 2379]
           etherT: ip
           prot: tcp
           stateful: no
        - name: blah
           range: [p1, p2]
           ...

Reply:
The formatting was lost in the comment. "range" should be aligned under "name", and similarly for other attributes.